packages/graphql/src/resolvers/Marketplace.js

```diff
@@ -58,5 +59,19 @@ export default {
     return events.length
   },
   users,
-  user: (_, args) => ({ id: args.id, account: { id: args.id } })
+  user: (_, args, context, info) => {
```

Comment:
Can we just return `null` if args.id isn't set?

Reply:
Should id ever not be set?  I have the same concerns as the other suggestion, that it wouldn't help diagnose.  But, I'm game if you want.

Reply:
we can keep in the console.log but we should never return { id: null }. Perhaps just add return null after line 73